front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -157,9 +158,11 @@ public Pipeline save(
     if (Strings.isNullOrEmpty(pipeline.getId())
         || (boolean) pipeline.getOrDefault("regenerateCronTriggerIds", false)) {
       // ensure that cron triggers are assigned a unique identifier for new pipelines
-      pipeline.getTriggers().stream()
-          .filter(it -> "cron".equals(it.getType()))
-          .forEach(it -> it.put("id", UUID.randomUUID().toString()));
+      Collection<Trigger> triggers = pipeline.getTriggers();
```

Comment:
We actually need to do this because `pipeline.getTriggers()` returns an ObjectMapper'd copy.